target/linux/realtek/files-5.10/include/dt-bindings/clock/rtl83xx-clk.h

```diff
@@ -0,0 +1,15 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
+/*
+ * Copyright (C) 2022 Markus Stockhausen
```

Comment:
I think upstream wants an e-mail address here generally

```suggestion
  * Copyright (C) 2022 Markus Stockhausen <markus.stockhausen@gmx.de>
```